doc/source/guidelines/logging.rst

```diff
@@ -1,62 +1,63 @@
 Logging Guidelines
 ##################
 
-This section describes several guidelines for logging in PyAnsys
-libraries. These guidelines are best practices discovered through
-implementing logging services and modules within PyAnsys
-libraries. Suggestions and improvements are welcome.
-External resources also describe `basic <https://docs.python-guide.org/writing/logging/>`__
-and `advanced <https://coralogix.com/blog/python-logging-best-practices-tips/>`__ technics.
+This section describes several guidelines for logging in PyAnsys libraries.
+These guidelines are best practices discovered through implementing logging
+services and modules within PyAnsys libraries. Suggestions and improvements are
+welcome.
+External resources also describe `basic
+<https://docs.python-guide.org/writing/logging/>`__ and `advanced
+<https://coralogix.com/blog/python-logging-best-practices-tips/>`__ technics.
 
 
 Description and usage
 =====================
-Logging helps to track events occurring in the application. For each of them a log record
-is created. It contains a detailed set of information about the current application operation.
-Whenever an information must be exposed, displayed and shared, logging is the
-way to do it.
-It is destinated to both the users and the application developers.
-It can serve several purposes:
+Logging helps to track events occurring in the application. For each of them a
+log record is created. It contains a detailed set of information about the
+current application operation. Whenever an information must be exposed,
+displayed and shared, logging is the way to do it.
+It is destinated to both the users and the application developers. It can serve
+several purposes:
 
   - extract some valuable data for the final users to know the status of their work.
   - track the progress and the course of the application usage.
   - provide the developer with as much information as possible if an issue happens.
 
-The message logged can contain generic information or embed data specific
-to the current session.
+The message logged can contain generic information or embed data specific to the
+current session.
 Message content is associated to a level of severity (info, warning, error...).
 Generally, this degree of significance indicates the recipient of the message.
-An info message is directed to the user while a debug message is useful for
-the developer itself.
+An info message is directed to the user while a debug message is useful for the
+developer itself.
 
 
 Logging in PyAnsys Libraries
 ============================
 
-The logging capabilities in PyAnsys libraries should be built upon the
-`standard logging <https://docs.python.org/3/library/logging.html/>`__
```

Comment:
Sorry for fixing something and reformatting at the same time.  This is the broken link, I've fixed it below (removed the unecessary / char).

Reply:
Thanks @dnwillia-ansys for fixing it.